docs/user-guide.md

```diff
@@ -37,13 +37,18 @@ This guide provides detailed instructions on how to import and configure your Li
 
     - Liberty Maven Plugin -> 3.7.1
 
-    - Liberty Gradle Plugin -> 3.5.1
+    - Liberty Gradle Plugin -> 3.6.1
```

Comment:
Noting that this change for the recommended minimum is to support Gradle 8.x, which LGP 3.5.x did not support.

Reply:
Nice catch. I'll also upgrade LMP to 3.8.1